File: Basic/Java/src/samples/SdsInterpolationMode.java
package samples;

public enum SdsInterpolationMode {

    Continuous(0),
    StepwiseContinuousLeading(1),
    StepwiseContinuousTrailing(2),
    Discrete(3);
    private final int SdsInterpolationMode;

    private SdsInterpolationMode(int id) {
        this.SdsInterpolationMode = id;
    }

    public int getValue() {
        return SdsInterpolationMode;
    }
}

File: ocs_samples/library_samples/Python3/ocs_sample_library_preview/SDS/SdsInterpolationMode.py

# SdsInterpolationMode.py
#

from enum import Enum

class SdsInterpolationMode(Enum):
    """
    SdsInterpolationMode 0 -3
    """
    Continuous = 0	            # The default InterpolationMode is Continuous. 
                                    # Interpolates the data using previous and next index values
    StepwiseContinuousLeading = 1   # Returns the data from the previous index
    StepwiseContinuousTrailing	= 2 # Returns the data from the next index
    Discrete = 3                    # Returns 'null'
